src/components/SearchBar.tsx

```diff
@@ -12,80 +12,73 @@ import SearchBarNativeComponent, {
 } from '../fabric/SearchBarNativeComponent';
 
 export const NativeSearchBar: React.ComponentType<SearchBarProps> &
-  typeof NativeSearchBarCommands = SearchBarNativeComponent as any;
+  typeof NativeSearchBarCommands =
+  SearchBarNativeComponent as unknown as React.ComponentType<SearchBarProps> &
+    SearchBarCommandsType;
 export const NativeSearchBarCommands: SearchBarCommandsType =
-  SearchBarNativeCommands as any;
+  SearchBarNativeCommands as SearchBarCommandsType;
+
+type ViewRef = React.ElementRef<typeof NativeSearchBar>;
 
 type SearchBarCommandsType = {
-  blur: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
-  focus: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
-  clearText: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
-  toggleCancelButton: (
-    viewRef: React.ElementRef<typeof NativeSearchBar>,
-    flag: boolean
-  ) => void;
-  setText: (
-    viewRef: React.ElementRef<typeof NativeSearchBar>,
-    text: string
-  ) => void;
-  cancelSearch: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
+  blur: (viewRef: ViewRef) => void;
+  focus: (viewRef: ViewRef) => void;
+  clearText: (viewRef: ViewRef) => void;
+  toggleCancelButton: (viewRef: ViewRef, flag: boolean) => void;
+  setText: (viewRef: ViewRef, text: string) => void;
+  cancelSearch: (viewRef: ViewRef) => void;
 };
 
-class SearchBar extends React.Component<SearchBarProps> {
-  nativeSearchBarRef: React.RefObject<SearchBarCommands>;
-
-  constructor(props: SearchBarProps) {
-    super(props);
-    this.nativeSearchBarRef = React.createRef();
-  }
+function ImplSearchBar(props: SearchBarProps, ref: Ref<SearchBarCommands>) {
```

Comment:
Also, one thing about this line - should we go for `function` declaration, or a `const` as an arrow function? I saw we're going for an arrow function (e.g. in Screen.tsx component), but I'm leaving this into discussion 🤔

Reply:
I am going to refactor the Screen once again after this PR so that the code matches - there is an 'outer' arrow function and the 'inner' one is declared as here. I would leave it like this and apply changes to the Screen component